README.md

```diff
@@ -126,9 +122,9 @@ Set is a list of capabilities to test against. User can define as much capabilit
 
 In this case there will be 3 different variations (tasks) to run:
 
-1. Safari 44, Android 5.1, iOS 9.3.5 on iPad
-1. Opera 12, Android 5.1, iOS 9.3.5 on iPad
-1. Android 6.0 Browser, Android 5.1, iOS 9.3.5 on iPad
+1. Safari 44, Android 5.1 device, iOS 9.3.5 on iPad
```

Comment:
Android version now is 8. Do you think mentioning here android 5.1 and IOS 9.3.5 is adding some value?

Reply:
Removed all versions. Added latest version of chrome, as an example